toolkit/scss/_temporary-message.scss

```diff
@@ -7,18 +7,14 @@
 // The following placeholders must be present in your main stylesheet:
 // - shared_placeholders/_temporary-messages.scss
 
-%temporary-message-section {
-  @extend %temporary-message;
-  margin-bottom: $gutter;
-}
-
+%temporary-message-section,
 .temporary-message {
-  @extend %temporary-message-section;
+  @extend %temporary-message;
```

Comment:
I think the problem is that extend is being used and if the shared_placeholders.scss file is imported again further down by another file, the changes in the extended class will be overwritten.  This has already surfaced as a problem in the buyer app, where the shared sass placeholder partial was introduced after the temporary message partial.

I think the best thing is probably to leave it just now, but I will put a story in pivotal tracker for looking at how imports are being handled in general.  Even imports are left as they are, it probably is still worth putting in documentation to explain the decisions behind how they are being handled.